source/Government.cpp

```diff
@@ -273,6 +282,9 @@ void Government::Load(const DataNode &node)
 		friendlyDisabledHail = GameData::Phrases().Get("friendly disabled");
 	if(!hostileDisabledHail)
 		hostileDisabledHail = GameData::Phrases().Get("hostile disabled");
+
+	// Store this bool to not have to check IsEmpty all the time.
+	hasTravelRestrictions = travelRestrictions.IsEmpty();
```

Comment:
is this needed? wont the locationfilter just return false if nothing is loaded in it?

Reply:
Isempty looks for 10 things
This is an optimization

Reply:
#8162

Reply:
Alr great I'll remove it here then